app/src/main/java/org/wildstang/wildrank/androidv2/NoteBox.java

```diff
@@ -0,0 +1,42 @@
+package org.wildstang.wildrank.androidv2;
+
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import org.wildstang.wildrank.androidv2.activities.NotesActivity;
+
+/**
+ * Created by Liam on 2/21/2015.
+ */
+public class NoteBox
```

Comment:
This should probably implement ```View``` and have it inflate its own view and be its own item in the view hierarchy.